Source/CesiumRuntime/Private/Tests/CesiumSceneGeneration.cpp

```diff
@@ -0,0 +1,242 @@
+// Copyright 2020-2023 CesiumGS, Inc. and Contributors
+
+#if WITH_EDITOR
+
+#include "CesiumSceneGeneration.h"
+
+#include "Tests/AutomationEditorCommon.h"
+
+#include "GameFramework/PlayerStart.h"
+#include "LevelEditorViewport.h"
+
+#include "Cesium3DTileset.h"
+#include "CesiumGeoreference.h"
+#include "CesiumIonRasterOverlay.h"
+#include "CesiumSunSky.h"
+#include "GlobeAwareDefaultPawn.h"
+
+#include "CesiumTestHelpers.h"
+
+namespace Cesium {
+
+FString SceneGenerationContext::testIonToken(
+    "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiI3NjU3OGE4Zi0xOGM4LTQ4NjYtODc4ZS02YWNkMDZmY2Y1M2YiLCJpZCI6MjU5LCJpYXQiOjE2OTA4Nzg3MjB9.uxePYJL59S4pG5aqJHb9goikVSO-Px6xA7kZH8oM1eM");
+FString SceneGenerationContext::testGoogleUrl(
+    "https://tile.googleapis.com/v1/3dtiles/root.json?key=AIzaSyCnRPXWDIj1LuX6OWIweIqZFHHoXVgdYss");
```

Comment:
These keys are about to be revoked because they're 2 months old. So two requests here:
1. Update to the latest keys in the Sampes project.
2. Add a note to https://github.com/CesiumGS/cesium-unreal/blob/ue5-main/Documentation/release-process.md to remind us to update these keys with each release, as we already do in the Samples project.

Reply:
I've done these two steps, but also moved the google test to a branch with other tests too. #1249